content/blog/2022-01-31-january-22-community-gems.md

```diff
@@ -0,0 +1,190 @@
+---
+title: January '22 Community Gems
+date: 2022-01-31
+description: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month: DVC
+  Studio data, DVC for non-ML projects, getting started with CML, and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month: DVC
+  Studio data, DVC for non-ML projects, getting started with CML, and more.
+picture: 2022-01-31/jan-community-gems.png
```

Comment:
Since this image hasn't been added yet, it's causing the build to fail due to a bug in the home page. I'll open a pull request for fixing the bug! For now, if you replace this with an actual image the build should work!

Reply:
Thanks Julie! I added a placeholder image.

Reply:
Awesome! The fix has been merged to master as well :)